luixxiul/browser-laptop
# Normalize modal overlay dialogs (pre-PR)

TODO: add modal dialog to `about:styles`.

Normalize modal overlay dialogs.

Follow up to 8042

----

Test Plan 1:
1. npm run test -- --grep='Payments'
2. npm run test -- --grep='Sync Panel'

Test Plan 2:
1. Open about:preferences#payments
2. Click "Add funds"
3. Make sure the 3 icons inside the dialog body is aligned

Test plan 3:
1. Open about:preferences#payments
2. Click "Add funds"
3. Click "Fund with credit/debit"
4. Make sure the coinbase widget is displayed at the center

Test Plan 4:
1. Add your country code to `coinbaseCountries.js`
2. Open about:preferences#payments
3. Click "Add funds"
4. Click "Display QR code"
5. Make sure the 3 icons appear on the footer

Test Plan 5:
1. Open about:preferences#payments
2. Enable payments
3. Click "Add funds"
4. Click "Display QR Code"
5. Make sure both the QR image and the label is aligned at the center

Test Plan 6:
1. Enable coinbase widget (via VPN)
2. Open about:preferences#payments
3. Click "Add funds"
4. Make sure "Done" button is aligned with the other buttons
5. Disable the coinbase widget
6. Make sure "Done button" is aligned to the right side

Test Plan 7:
1. Open about:preferences#payments
2. Click "Add funds"
3. Click "Display QR code"
4. Hover on the QR code image
5. Make sure its title is displayed

Test Plan 8:
1. Open about:preferences#sync
2. Enable Sync
3. Click "Sync a new device"
4. Click "Show secret code words"
5. Make sure the code is selectable
